manifests/init.pp

```diff
@@ -225,10 +225,6 @@
     fail("Setting \$config_keys_manage false and \$chrony_password at same time in ${module_name} is not possible.")
   }
 
-  if $queryhosts != [] and $port == 0 {
-    fail("Setting \$queryhosts has no effect unless also setting \$port which defaults to 0 in ${module_name}, refusing that.")
-  }
```

Comment:
Should this be kept in but with `and !$port`?

Reply:
I don't think so. The point of this change is that I want to accept client access without needing to specify the port, which then is the default ntp-port 123. But as you said, this needs to be reviewed by someone more with chrony knowledge.

I also noticed that the unit-test failed. I need to also change the test. Unit-testing is a great thing! :)